Sources/Metrics/Docs.docc/index.md

```diff
@@ -0,0 +1,5 @@
+# ``Metrics``
+
+A Metrics API package for Swift.
+
+Refer to `CoreMetrics` module documentation for the majority of types.
```

Comment:
Ah, good idea to have an `index.md` in each module. 👍 

For me I would choose to have the detailed docs in `Metrics` instead because it matches the name of the package, but I'm ok either way.

Reply:
Hm... i guess better to keep "the docs" in the module which has the types since we can't link to the types otherwise -- the cross module limitation of docc -- 😢